test/pad_table.md

```diff
@@ -0,0 +1,26 @@
+# This is a test document
+
+With some text, `code`, **bolds** and _italics_.
+
+## This is second header
+
+Displaynone text
+
+Header 1  | Header 2  | Header 3                                     
+----------|-----------|----------------------------------------------
+Content 1 | Content 2 | ![200](http://lorempixel.com/200/200) Image! 
+Content 1 | Content 2 | ![200](http://lorempixel.com/200/200) Image! 
```

Comment:
Sounds good. I essentially copied the google_doc_tables test case and derived from there. I'm pushing this.